src/com/trollworks/gcs/character/TextTemplate.java

```diff
@@ -868,10 +868,17 @@ private String hitLocationEquipment(HitLocationTableEntry entry) {
                             // Assume Vitals uses the same equipment as Torso
                             locationKey = HitLocation.TORSO.getKey();
                         }
+                        boolean showEquipment = ((DRBonus) feature).getLocation() == com.trollworks.gcs.feature.HitLocation.FULL_BODY;
```

Comment:
Factor out `((DRBonus) feature).getLocation()` and use it where needed, rather than repeating it.